aatools/food_insecurity/fewsnet.py

```
@@ -0,0 +1,218 @@
+"""
+FEWS NET processing.
+
+Download and save the data provided by FEWS NET.
+"""
+
+import logging
+from datetime import datetime
+from pathlib import Path
+
+from aatools.utils.io import download_url, unzip
+
+logger = logging.getLogger(__name__)
+
+
+def download_zip(
+    url: str,
+    zip_path: Path,
+    output_dir: Path,
+) -> bool:
+    """
+    Download and unzip the file at the url.
+
+    Parameters
+    ----------
+    url : str
+        url that contains the zip file to be downloaded
+    zip_path : Path
+        path to which the content of the url should be downloaded
+    output_dir : Path
+        path of dir to which the zip content should be written
+
+    Returns
+    -------
+    valid_file: bool
+        if True, the url contains a valid zip file
+
+    """
+    valid_file = False
+
+    try:
+        download_url(url, zip_path)
+        logger.info(f"Downloaded {url} to {zip_path}")
+
+        try:
+            unzip(zip_path, output_dir)
+            logger.debug(f"Unzipped {zip_path}")
+            valid_file = True
+        except Exception:
+            # indicates that the url returned something that wasn't a
+            # zip, happens often and indicates data for the given
+            # country - date is not available
+            logger.debug(f"No zip data returned from url {url}")
+        # remove the zip file
+        zip_path.unlink()
+
+    except Exception:
```

Same as above, could a catch-all exception be avoided?